server/conf/conf.go

```diff
@@ -157,6 +163,12 @@ type NATSStreamingConfig struct {
 	ConnectWait        int // milliseconds
 }
 
+// JetStreamConfig configuration for a JetStream connection
+type JetStreamConfig struct {
+	PublishAsyncMaxPending int
+	MaxWait                int // milliseconds
```

Comment:
EnableFlowControl and maybe heartbeats as well might be useful, this sounds like similar to how mirrors/sources work

Reply:
Added. 👍🏽